src/App.css

```diff
@@ -1,38 +1,39 @@
-.App {
-  text-align: center;
+#root {
+  width: 700px;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+input {
+  background: gray;
+  color: white;
+  font-weight: bold;
+  grid-column-start: 1;
+  grid-column-end: 5;
+  padding: 10px;
+  height: 100px;
+  width: 676px;
+  text-align: right;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+button {
+  color: black;
+  font-weight: bold;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.grid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-rows: 100px;
+}
+
+.double {
+  grid-column-start: 1;
+  grid-column-end: 3;
 }
 
-.App-link {
-  color: #61dafb;
+.orange {
+  background-color: orange;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.simple {
+  background-color: lightcyan;
```

Comment:
### [optional]
Since there's enough space in these buttons, you might want to increase the size of the button text
```
font-size: 2em;
```